protocols/cc_protocol.py

```diff
@@ -53,18 +52,22 @@ def get_figure2_cmds(self, mm_delay, uplink_trace, downlink_trace, args):
         """ Returns ordered dictionary of commands
         to run to generate Figure 2 results.
         """
-        
+        FLIP_FULL = True
+        link = self.config.get('link', 'uplink')
         if self.config['uplink_queue'] == '':
             queue_args = ''
         else:
             queue_args = self.mahimahi_queue_args_fmt.format(
-                    uplink_queue=self.config['uplink_queue'], 
-                    uplink_queue_args=self.config['uplink_queue_args']
+                    link=link if FLIP_FULL else 'uplink',
```

Comment:
This can just be link

Reply:
I don't think so?

Reply:
ehh, ok I trust you on this.  As long as it's fully compatible with all the other experiments